scipy/fftpack/_helper.py

```diff
@@ -1,7 +1,10 @@
 import operator
-from numpy.fft.helper import fftshift, ifftshift, fftfreq
+
```

Comment:
I'm not sure we should be introducing any functionality changes to the legacy `fftpack` module. It would be better to just fix the import.

```suggestion
from numpy.fft import fftshift, ifftshift, fftfreq
```

Reply:
I agree. `numpy.fft` imports `from .helper import *` so it guarantees the same behavior.